### ISSUE TITLE
How to send commands programatically?

### ISSUE THREAD

**Issue by hackerunet** [tananaev/traccar#3042]
Hi there Anton, I've been looking for all information about GPRS commands and there are few clues about it, I would like to know exactly how can I send a command programatically without using the traccar interface, how exactly does this traccar server?
Thanks.

**Comment by tananaev** [comment]
All information is already available. You can:

1. Read the API documentation
2. Check source code
3. Check what official app sends
4. Find example on GitHub or forum

It has been asked many times before and there are some examples, and I think even code snippets that you can use.

**Comment by hackerunet** [comment]
ok I'm not subscribed in forums. I'll do it, thanks a lot!!!...

**Comment by tananaev** [comment]
You don't need to subscribe to forums. You can just search it. I recommend just Google for the answer.

**Comment by hackerunet** [comment]
Sorry to answer this, I found this: https://github.com/tananaev/traccar/issues/2023
I have exactly the same issue, I need to know what queries do I have to prepare and support in the config file to be able to send a command via GPRS, I have also found through forums more information, I can see that I need to enable the default web interface to be able to launch the API server. I have an nginx web server, is it possible to enable the API using my nginx server or do I need to enable the default web interface ?
Also, is there any method to build a commands list queue to send to the devices? what I mean is, if I have full database access can I just add a record in some table and make traccar to lookup for records in that table and start processing all commands in it? I do this for SMS and loggin because I need to know what user is sending certain commands.
Thanks
